slides/02-deploy.qmd

```diff
@@ -263,7 +272,9 @@ jsonlite::fromJSON(metadata)
 
 ```{python}
 #| eval: false
-board.pin_meta("isabel.zimmerman/test_model").user
+import requests
+url = "https://colorado.posit.co/rsc/chicago-inspections-python/metadata" 
+print(requests.get(url).content)
```

Comment:
I did update this requests call, since I was going through anyway. When this PR gets merged, I'll close out #8